bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/Adapters.java

```diff
@@ -62,6 +63,8 @@
           new BigtableWhileMatchResultScannerAdapter(FLAT_ROW_ADAPTER);
   /** Constant <code>GET_ADAPTER</code> */
   public static final GetAdapter GET_ADAPTER = new GetAdapter(SCAN_ADAPTER);
+  /** Constant <code>PROTO_ROW_ADAPTER</code> */
+  public static final ProtoRowAdapter PROTO_ROW_ADAPTER = new ProtoRowAdapter();
```

Comment:
I don't think this belongs here. Conversion to Result should be private to the `DataClientWrapper`. `Adapters` is a layer above